packages/container/lib/polyfills.ts

```diff
@@ -0,0 +1,14 @@
+// Polyfill for :focus-visible pseudo-selector
+// See: https://github.com/WICG/focus-visible
+import "focus-visible"
+
+// Standard base stylesheet used across Culture Amp products
+// See: https://github.com/necolas/normalize.css/
+import "normalize.css"
+
+// Global styles
+import { } from "./globals.module.scss"
+
+// Supported / active face declarations
+import { } from "./fonts.module.scss"
```

Comment:
Nitpick: I feel like these two should maybe be imported from the main `index.tsx` file, since these aren't really polyfills.